source/wp-content/themes/wporg-news-2021/sass/components/footer-archive/_footer-archive.scss

```diff
@@ -1,32 +1,47 @@
 // Default/light version.
-.footer-archive .footer-archive-row[class*="wp-container-"] {
-	justify-content: space-between;
-	align-items: stretch !important;
-	margin-top: 0;
-	gap: 0;
-	background-color: var(--wp--preset--color--white);
-
-	& > * {
-		padding-top: 50px;
-		padding-bottom: 50px;
-		border-bottom: 1px solid var(--wp--preset--color--light-grey);
-
-		@include break-medium {
-			flex-basis: 50%;
-			flex-shrink: 0;
-			flex-grow: 0;
-			padding-top: 60px;
-			padding-bottom: 60px;
-			border-top: 1px solid var(--wp--preset--color--light-grey);
-			border-bottom: none;
-
-			&:nth-child(odd) {
-				border-right: 1px solid var(--wp--preset--color--light-grey);
-				padding-right: var( --wp--custom--alignment--edge-spacing );
-			}
+.footer-archive {
+	h2 {
+		display: none;
```

Comment:
`display: none` actually hides this content from screen readers too. You need to do [a bit more to visually hide content](https://github.com/WordPress/gutenberg/blob/3475693ae69d3464defa842a76da8505bc275a46/packages/block-library/src/common.scss#L62), which is why the `screen-reader-text` class helper exists 🙂

Reply:
This `display:none` still needs to be removed, it's hiding the heading:

<img width="301" alt="Screen Shot 2021-12-08 at 10 36 53 AM" src="https://user-images.githubusercontent.com/541093/145236887-6f44da63-a856-49e0-9131-1bc0cf2a67c5.png">